Fix a FTBFS with libpoppler 0.11: the destructor of GfxFont is virtual, and it doesn't make sense to call it.
Taken from Ubuntu
---
 source/texk/web2c/luatexdir/image/pdftoepdf.cc |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: luatex-0.47.0/source/texk/web2c/luatexdir/image/pdftoepdf.cc
===================================================================
--- luatex-0.47.0.orig/source/texk/web2c/luatexdir/image/pdftoepdf.cc	2009-12-18 18:37:54.000000000 +0900
+++ luatex-0.47.0/source/texk/web2c/luatexdir/image/pdftoepdf.cc	2009-12-19 18:26:42.000000000 +0900
@@ -703,7 +703,7 @@
     }
     for (r = encodingList; r != NULL; r = n) {
         n = r->next;
-        delete r->font;
+//        delete r->font;
         delete r;
     }
 }
